lib/highlighter.js

```diff
@@ -5,6 +5,27 @@ var blacklist = require('blacklist');
 var createReactClass = require('create-react-class');
 var PropTypes = require('prop-types');
 
+function removeDiacritics(str, blacklist) {
+  if (!String.prototype.normalize) {
```

Comment:
This pretty much excludes IE. [This solution](https://stackoverflow.com/questions/990904/remove-accents-diacritics-in-a-string-in-javascript/18391901#18391901) is more widely cross-browser compatible.